docs/lib/auth/fragments/android/user_attributes/10_fetch_attributes.md

```diff
@@ -12,10 +12,12 @@ Amplify.Auth.fetchUserAttributes(
 <amplify-block name="Kotlin">
 
 ```kotlin
-Amplify.Auth.fetchUserAttributes(
-    { Log.i("AuthDemo", "User attributes = $it") },
-    { Log.e("AuthDemo", "Failed to fetch user attributes.", $it) }
-)
+try {
+    val attributes = Amplify.Auth.fetchUserAttributes()
+    Log.i("AuthDemo", "User attributes = $attributes")
```

Comment:
Thanks for raising this point, @Kilo-Loco! I think it does make sense to go through and align the various tags. Doing so will require me touch the _other_ code snippets for Java and RxJava, though. So, I'm thinking it would be better to tackle this in a separate PR. I did update the other ones that said `"TAG"`, since that was pretty egregious. I'll do a follow-up PR to get these `"AuthDemo"` and `"MyAmplifyApp"` tags to be more consistent with their surroundings.

Reply:
Awesome. Makes sense to me 👍🏽